rollup.config.js

```diff
@@ -17,7 +18,8 @@ export default [
       eslint(),
       copy(
         [
-          { files: 'profiles/**', dest: `${DIST_FOLDER}/profiles` }
+          { files: 'profiles/**', dest: `${DIST_FOLDER}/profiles` },
+          { files: 'node_modules/three/**', dest: `${VIEWER_FOLDER}/three` }
```

Comment:
I guess we could copy just the required files if you'd like.

Reply:
Since it's just going to gh-pages I guess I don't care that much. Just makes the build noisier, but it would be more error prone to have to maintain the list of files that are in use.

Reply:
really noisy.  Perhaps instead we can add a second copy of the plugin that isn't verbose for these files?